packages/store/package.json

```diff
@@ -8,43 +8,57 @@
     "directory": "packages/store"
   },
   "license": "MIT",
+  "type": "module",
   "exports": {
-    "./mud.config.mjs": "./dist/mud.config.mjs",
-    "./*": "./*"
+    ".": "./dist/index.js",
+    "./*": "./dist/*",
+    "./abi/*": "./abi/*"
   },
-  "types": "./types/ethers-contracts/",
+  "types": "ts/index.ts",
   "scripts": {
-    "build": "pnpm codegen && rimraf out && forge build && pnpm dist && pnpm types && tsup",
-    "codegen": "tsx ./scripts/codegen.ts && prettier --write '**/*.sol'",
-    "dist": "rimraf abi && mkdir abi && cp -rf out/*.sol/*.json abi/ && pnpm rimraf abi/*.metadata.json",
+    "build": "pnpm run build:js && pnpm run build:tightcoder && pnpm run build:codegen && pnpm run build:abi && pnpm run build:typechain",
+    "build:abi": "forge build --out abi --skip test script",
+    "build:codegen": "node ./dist/tablegen.js",
```

Comment:
should we use `tsx` and the source here?

Reply:
done in https://github.com/latticexyz/mud/pull/640/commits/77487c03bc674acebf1d7501810259c128e4a256